mouse-look.js

```diff
@@ -43,6 +45,14 @@ WL.registerComponent('mouse-look', {
             }
         }.bind(this));
 
+        if(this.pointerLockOnClick){
+            WL.canvas.addEventListener('mousedown', function() {
+                WL.canvas.requestPointerLock = WL.canvas.requestPointerLock ||WL.canvas.mozRequestPointerLock ||WL.canvas.webkitRequestPointerLock;
+                // Ask the browser to lock the pointer
```

Comment:
Throughout the code-base we have a rather unusual convention of using `//` only for temporary comments during development and use `/* */` for permanent comments. Sorry for the nit-pick 😌 

In this case, though, I think the following line is pretty clear, so probably no comment necessary.